ansible/helm-release.yml

```diff
@@ -97,6 +97,7 @@
           command:
             cmd: "{{ item }}"
           loop:
+            - git config --global url.https://{{ gh_user }}:{{ gh_token }}@github.com/.insteadOf https://github.com/
```

Comment:
Maybe make this its own task with no_log just incase the user/token get spit out in stdout/stderr.